core/tests/test_utils.py

```diff
@@ -183,8 +183,8 @@ def get_storage_model_module_names():
     """Get all module names in storage."""
     # As models.NAMES is an enum, it cannot be iterated over. So we use the
     # __dict__ property which can be iterated over.
-    for name in models.NAMES.__dict__:
-        if '__' not in name:
+    for name in models.NAMES:
+        if '__' not in name.value:
```

Comment:
I think that the dict might have been needed only in the previous implementation, can you check it please?

Reply:
Yes, it was required only for the previous implementation. Will confirm though